src/commonMethods/webKitPerformanceCommonFunctions.js

```diff
@@ -0,0 +1,61 @@
+let samples = []
+
+/**
+ * This function resets the Samples array
+ */
+export const initializeSamplesArray = function() {
+  samples = []
+}
```

Comment:
instead of `samples`-array and `initializeSamplesArray` consider using `this.$data` to store data in a test (https://github.com/WebPlatformForEmbedded/Storm/blob/master/docs/features/data.md)

Reply:
Updated as per the code review comment in the latest commit